command.go

```diff
@@ -233,6 +206,77 @@ func (c Command) Run(ctx *Context) (err error) {
 	return err
 }
 
+func getIndexes(ctx *Context) (int, int){
+	firstFlagIndex := -1
+	terminatorIndex := -1
+	for index, arg := range ctx.Args() {
+		if arg == "--" {
+			terminatorIndex = index
+			break
+		} else if arg == "-" {
+			// Do nothing. A dash alone is not really a flag.
+			continue
+		} else if strings.HasPrefix(arg, "-") && firstFlagIndex == -1 {
+			firstFlagIndex = index
+		}
+	}
+	if len(ctx.Args()) > 0 && !strings.HasPrefix(ctx.Args()[0], "-") && firstFlagIndex == -1{
+		return -1, -1
+	}
+
+	return firstFlagIndex, terminatorIndex
+
+}
+
+func copySlice(slice []string, start, end int) []string{
+	var newSlice []string
```

Comment:
I think something like

```
newSlice := make([]string, end - start)
copy(newSlice, slice[start:end)
return newSlice
```

would be a bit more performant. Maybe also call this `copyStringSlice`?

Reply:
Ok fixed